src/components/Switzerland/Map.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { geoPath, geoTransform } from 'd3-geo';
+import { scaleLinear } from 'd3-scale';
+import ReactTooltip from 'react-tooltip';
+import { TimeZipCodeDistributionEntry } from '../../services/api-types';
+import { scaleQuantile } from 'd3-scale';
+import styled from 'styled-components';
+
+import bbox from '@turf/bbox';
+import relief from './relief.jpg';
+import geoJson from './PLZ10.json';
+
+const PADDING_SCALE = 1;
+const WIDTH_ADJUST = 16 * 2 * PADDING_SCALE;
+
+const Wrapper = styled.div`
+  padding: ${PADDING_SCALE}rem ${PADDING_SCALE}rem ${PADDING_SCALE}rem 1rem;
```

Comment:
You are adding padding in `Map`, but doing measurement in `Switzerland`. I think it will simplify the code if you add the padding in the same component where you are measuring.

Specifically, if you add padding in `Switzerland` instead of `Map`, then you can get rid of most of the padding code (`WIDTH_ADJUST`, `position: relative`, etc). The trick is that `ResizeObserver` (which `react-resize-detector` uses) can measure `content-box` (actually this is the default; [doc](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver/observe)). So if you wrap your `Map` in a `div` in `Switzerland`, put the padding on that `div`, and measure it, then the measurements will be without padding.

Reply:
- [ ] Move padding to object where measurement is being done